packages/material-ui-lab/src/Autocomplete/Autocomplete.test.js

```diff
@@ -75,6 +75,56 @@ describe('<Autocomplete />', () => {
       fireEvent.change(document.activeElement, { target: { value: 'o' } });
       checkHighlightIs('one');
     });
+
+    it('should set the focus on selected item when dropdown is expanded', () => {
```

Comment:
@oliviertassinari I'm not sure if this test is in right place

Reply:
Me too, I think that it's a generic feautre the component should have.